### ISSUE TITLE
Spectronaut template not working

### ISSUE THREAD

**Issue by educhicano** [ftwkoopmans/msdap#13]
Hi! Congrats for your amazing work and the easy to use package. 
I'm using a short experiment with 6 cell culture DIAPASEF. During the experiment I've used both Spectronaut and DIANN.
I've downloaded your report template for Spectronaut, but during the analysis, dplyr package gives me an error. Not only for this test experiment,  other experiments shown the same problems. 
Interestingly, if I load the data from DIANN there's no problem at all.
Attached I send a screen capture of console showing the error when working with spectronaut:

<img width="1170" alt="Captura de pantalla 2023-05-30 a las 13 33 08" src="https://github.com/ftwkoopmans/msdap/assets/21240536/bf526f07-c3c2-48ec-8e8e-3fb9506692fd">

**Comment by ftwkoopmans** [comment]
Looking at the log, it seems that there are no peptides in the dataset that are detected across all sample groups at given filtering settings. From your description, the dataset has 6 samples (I assume 3 per condition) and the log shows you required "min_detect=3" in the MS-DAP filtering settings; this resulted in "0/121700 peptides were retained after filtering over all groups".

This is unusual. Either the peptide/precursor confidence scores estimated by Spectronaut ("EG.Qvalue" column in the Spectronaut report) are quite low in general, or there are no peptides that have a good "EG.Qvalue" across all samples according to Spectronaut (perhaps there is 1 outlier sample with overall low quality scores?).

You could first try to run MS-DAP with "min_detect=2" instead and see if that does yield a substantial number of peptides that is identified across sample groups. If that doesn't work, feel free to privately share your dataset with me by email (address @ corresponding author in the MS-DAP paper) and I'll double-check that this is an issue with your Spectronaut report/data and not MS-DAP (and in case of an MS-DAP bug, I'll issue a bugfix asap).

**Comment by ftwkoopmans** [comment]
Thanks again for your bug report and the data you shared via email.

I've just released [version 1.0.5](https://github.com/ftwkoopmans/msdap/releases/tag/1.0.5) that should address this issue. To update;
- close RStudio and then start a new RStudio session
- run the following code snippet to download and update MS-DAP
```
Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS="true")
devtools::install_github("ftwkoopmans/msdap", upgrade = "never")
```

Please let me know if it all works out.

**Comment by educhicano** [comment]
Hi Frank, 
now it seems to work again. Thanks for your help!
Best regards,
Edu
